src/metabase/lib/metadata/calculation.cljc

```diff
@@ -466,15 +466,20 @@
   [query _stage-number stage-number options]
   (returned-columns-method query stage-number (lib.util/query-stage query stage-number) options))
 
-(def ^:dynamic *propagate-inherited-temoral-unit*
-  "Enable propagation of ref's `:temporal-unit` into `:inherited-temporal-unit` of a column.
+(def ^:dynamic *propagate-options-to-next-stage*
```

Comment:
I have the feeling this name suggests something more general than what it does. At least currently, it seems to be limited to propagating binning and bucketing info. I suppose we should have a common name for these two things and should probably include it in the name of this var.

Reply:
I had a same feeling but was hesitant introducing new term. Maybe `grouping`? But that is probably also too general. WDYT?

I've changed the name of the var to `*propagate-binning-and-bucketing*` to make it more specific for the time being. I expect to be doing a follow-up PR due to further comments on MLv2 functions. If you are fine with grouping or have a better idea, I'd like to address that in the next PR.

Reply:
I think grouping would be really misleading (it's used for something different). `*propagate-binning-and-bucketing*` seems OK.